### PR TITLE
Added Splash Screen

### PR DESCRIPTION
**Pull request by R0gue-one**
Created a draft for the Splash screen. Added app logo and tagline. 

I don't know the font for the tagline so just have used the default font will update ASAP

Resolving issue: https://github.com/AmanNegi/FreshNest/issues/143

### CODE REVIEW

**Comment by AmanNegi** [comment]
Could you share a demo video real quick?

**Comment by AmanNegi** [comment]
Everything so far looks amazing! Just could you use the app accent color for text. Also avoid pushing `pubspec.lock` changes.

**Comment by R0gue-one** [comment]
@AmanNegi Sorry for the delay. I have now changed the text color of tagline. Here the preview:

Screenshot_20231016_231735


If you have any suggestions or anything you want to change please let me know.

**Comment by R0gue-one** [comment]
Also sorry for pushing the changes in `pubsec.lock` I didn't realise it that time, I will keep that in mind now

File: frontend/flutter/lib/splash.dart [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  @override
+  void initState() {
+    super.initState();
+    _naviagteToHome();
+  }
+
+
+  _naviagteToHome()async{
+    await Future.delayed(Duration(milliseconds: 1700), () {});
+    Navigator.pushReplacement(context, MaterialPageRoute(builder: (context)=>App().getHomePage()));
```
Comment (AmanNegi) [inline frontend/flutter/lib/splash.dart:22]: Feel free to move the `getHomePage` function in this file. Also move the RolePage widget.

File: frontend/flutter/lib/splash.dart [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        const Align(
+          alignment: Alignment.bottomCenter,
+          child: Padding(
+            padding: EdgeInsets.only(bottom: 200.0),
+            child: Text(
+              "Your cozy corner \nfor farm fresh \n delights!",
+              textAlign: TextAlign.center,
+              style: TextStyle(
+                fontSize: 20.0,
+                //rgba(4,141,66,255)
```
Comment (AmanNegi) [inline frontend/flutter/lib/splash.dart:49]: Remove the commented code.

File: frontend/flutter/lib/splash.dart [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                color: Color.fromARGB(255,4,141,66),
```
Comment (AmanNegi) [inline frontend/flutter/lib/splash.dart:50]: We have a colors.dart file, you can use the accent color variable here.

**Review by AmanNegi — CHANGES_REQUESTED** [review]
Here's some changes.

**Review by AmanNegi — APPROVED** [review]

**Comment by AmanNegi** [comment]
@R0gue-one great work, can you remove the `pubspec-lock` changes from this PR? Or should I do it.

It's as simple as downloading the current `pubspec.lock` from the `main` branch and replace it with yours. Then simply commit this change and push it.

**Comment by R0gue-one** [comment]
> @R0gue-one great work, can you remove the `pubspec-lock` changes from this PR? Or should I do it.
> 
> It's as simple as downloading the current `pubspec.lock` from the `main` branch and replace it with yours. Then simply commit this change and push it.

Sure I'll do it right away

**Comment by R0gue-one** [comment]
Done !

**Comment by AmanNegi** [comment]
Looks great! Thanks a lot for contributing to FreshNest. I will merge your PR right away. :smile_cat: :rocket:

**Comment by R0gue-one** [comment]
Thank you very much having me @AmanNegi and helping me though it !!

**Comment by AmanNegi** [comment]
Feel free to come back for more issues. :sparkles:

**Comment by R0gue-one** [comment]
Sure will !

**Comment by R0gue-one** [comment]
Hey! @AmanNegi there was one thing that I wanted to mention I had a problem while setting up the project, I think we need to add .json file from google services? I didn't know about this as I just started flutter, should we add this to the Readme file under flutter ?

**Comment by AmanNegi** [comment]
Yes, @R0gue-one we should add this in Readme. Feel free to create an issue and raise a PR as well.

**Comment by R0gue-one** [comment]
Sure I'll get on it !

**Pull request merged**